types/evse_security.yaml

```diff
@@ -183,13 +191,38 @@ types:
           minimum: 0
           type: object
           $ref: /evse_security#/OCSPRequestData
-  KeyPair:
+  CertificateOCSP:
+    description: OCSP data related to requested the certificates
+    type: object
+    required:
+      - hash
```

Comment:
Because if we have a chain of returned certificates, for example leaf<-subca2<-subca1, we agreed that a list of the same count of OCSP data will be returned, because matching the ocsp data to a certificate in everest-core will be very difficult, based on a certificate hash, because it is difficult to build that has, especially when intermediaries are involved. So, if a chain of 3 certificates is returned, then a vector of 3 ocsp data is returned, each matched to the certificate from the chain in order.